includes/cognitive-services-speech-service-endpoints-speech-to-text.md

```diff
@@ -12,7 +12,7 @@ Region|	Speech to Text endpoint
 West US| `https://westus.stt.speech.microsoft.com/speech/recognition/conversation/cognitiveservices/v1`
 West US2| `https://westus2.stt.speech.microsoft.com/speech/recognition/conversation/cognitiveservices/v1`
 East US| `https://eastus.stt.speech.microsoft.com/speech/recognition/conversation/cognitiveservices/v1`
-East US2| `https://eastus.stt.speech.microsoft.com/speech/recognition/conversation/cognitiveservices/v1`
+East US2| `https://eastus2.stt.speech.microsoft.com/speech/recognition/conversation/cognitiveservices/v1`

```

Comment:
don't know when/if it was changed, but on the doc side I see the correct link
[https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/rest-apis](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/rest-apis)